scripts/install-cni.sh

```diff
@@ -50,6 +51,24 @@ echo "Install-CNI ($0), Build: $BUILD"
 
 set -u -e
 
+# Try to decouple RUN_CNI_WATCHDOG and ENABLE_CILIUM_PLUGIN; don't assume
+# ENABLE_CILIUM_PLUGIN is set whenever RUN_CNI_WATCHDOG is set.
+# All exit paths should call exit_success instead of `exit 0`,
+# so the appropriate actions can be taken according to RUN_CNI_WATCHDOG.
+exit_success() {
```

Comment:
rename ? 

We do not exit when watchdog is enabled

Reply:
Do you have a recommendation? I'm not good at doing naming...

Reply:
Done